resources/pipeline/rackhd/ova/stack_config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "ova": {
+    "rackhd_host": "172.31.128.1",
```

Comment:
The ip of ova is fixed to "172.31.128.1" ?

Reply:
The real IP of ova is 172.31.128.1